src/header/classes.scss

```diff
@@ -1,11 +1,73 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
+@import '../breadcrumbs/index';
 
 .iui-page-header {
   @include iui-page-header;
+  @include iui-page-header-slim;
 }
 
-.iui-header-logo {
-  @include iui-header-logo;
+.iui-page-header-left,
+.iui-page-header-center,
+.iui-page-header-right {
+  @include iui-page-header-zone;
+}
+
+.iui-page-header-right {
+  @include iui-page-header-right;
+}
+
+.iui-page-header-divider {
+  @include iui-page-header-divider;
+}
+
+.iui-header-brand {
+  @include iui-header-brand;
+}
+
+.iui-header-brand-icon {
+  @include iui-header-brand-icon;
+}
+
+.iui-header-brand-label {
+  @include iui-header-brand-label;
+}
+
+.iui-header-breadcrumbs {
+  @include iui-breadcrumbs($breadcrumbs-height: 100%);
```

Comment:
I don't think `classes.scss` should have any logic so I suggest creating a new `@mixin iui-header-breadcrumbs` which handles this.

Reply:
Can do, change made.